Package@swift-5.2.swift

```diff
@@ -20,9 +20,14 @@ let package = Package(
       url: "https://github.com/apple/swift-argument-parser.git",
       .upToNextMinor(from: "0.3.0")
     ),
+    .package(
+      name: "SwiftPM",
+      url: "https://github.com/apple/swift-package-manager.git",
+      .branch("main")
```

Comment:
They've just created the 5.4 branch yesterday, maybe it's worth relying on that one now instead of 5.3? Usually, when a release branch is created, they don't add much breaking changes to it if any, so probably worth trying it out.

```suggestion
      .branch("release/5.4")
```

Reply:
Especially, as this PR already uses `main`, `release/5.4` shouldn't differ much from it.